system-helper/flatpak-system-helper.c

```diff
@@ -1806,18 +1807,19 @@ authorize_deploy_add_polkit_details (const gchar           *installation,
   g_autoptr(GError) local_error = NULL;
   g_autoptr(FlatpakDir) system = NULL;
   g_autoptr(AsApp) app = NULL;
-  g_autoptr(EpcAppFilter) app_filter = NULL;
+  g_autoptr(MctManager) manager = NULL;
+  g_autoptr(MctAppFilter) app_filter = NULL;
   g_autoptr(AutoPolkitSubject) subject_process = NULL;
   gint subject_uid;
 
-  /* The ostree-metadata and appstream/* branches should not have any parental
+  /* The ostree-metadata and appstream/ branches should not have any parental
    * controls restrictions. */
```

Comment:
Spurious change?

Reply:
IIRC it’s a copy of a change I did in the upstream MR to stop the compiler complaining about `/*` being present inside a comment. I don’t think it’s spurious, although it should be in a separate commit (if it isn’t in a separate commit in the upstream MR, then that’s my bad).

Reply:
Yes, you are correct, I will push it in a separate commit - iirc upstream doesnt have it separate but I think that is fine cause the code wasnt there before.